cluster/addons/ip-masq-agent/ip-masq-agent.yaml

```diff
@@ -21,6 +21,13 @@ spec:
       containers:
       - name: ip-masq-agent
         image: gcr.io/google-containers/ip-masq-agent-amd64:v2.0.1
+        resources:
+          limits:
+            cpu: 100m
```

Comment:
I wonder if we hit the iptables 10x slowdown with this, how bad it will be.  what if we had no limit for CPU?